api/internal/infrastructure/mongo/container.go

```
@@ -31,7 +31,7 @@
 		Asset:       NewAsset(client),
 		AuthRequest: authserver.NewMongo(client.WithCollection("authRequest")),
 		Config:      NewConfig(db.Collection("config"), lock),
-		Deployment:  NewDeployment(client),
+		Deployment:  NewDeployment(client).(repo.Deployment),
```

golangci-lint (1.62.2)

S1040: type assertion to the same type: NewDeployment(client) already has type repo.Deployment

(gosimple)